### PR TITLE
doc: ctc meeting minutes 2017-06-07

### PR DESCRIPTION
**Pull request by joshgav**
cc @nodejs/ctc

### CODE REVIEW

**Review by Trott — APPROVED** [review]
LGTM

**Review by evanlucas — APPROVED** [review]

**Review by mcollina — APPROVED** [review]
LGTM 🎉

**Review by jasnell — APPROVED** [review]

File: meetings/2017-06-07.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+### doc: note that EoL platforms are not supported [#12672](https://github.com/nodejs/node/pull/12672)
+ 
+@mhdawson: suggestion to add to support policy that platforms are no longer supported once they reach end-of-life, even if Node.js hasn't changed versions.
+ 
+Question is what does support mean to begin with - is it just "best effort"? And what does it mean that it changes if the platform isn't supported?
+ 
+@mylesborins: A good example is Node 6 and Ubuntu 12.04. That version of Ubuntu isn't supported by Canonical, should we continue to support Node 6 on it?
+ 
+@mhdawson: If we tell people from the beginning of the semver-major that support for unsupported platforms will cease even within semver-major cycle, that would be okay.
+
```
Comment (mhdawson) [inline meetings/2017-06-07.md:60]: "Some of the discussion in the issue is that if we tell people from the beginning of the semver-major that support for unsupported platforms will cease even within semver-major cycle, that would be okay.

File: meetings/2017-06-07.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+ 
+@mhdawson: suggestion to add to support policy that platforms are no longer supported once they reach end-of-life, even if Node.js hasn't changed versions.
+ 
+Question is what does support mean to begin with - is it just "best effort"? And what does it mean that it changes if the platform isn't supported?
+ 
+@mylesborins: A good example is Node 6 and Ubuntu 12.04. That version of Ubuntu isn't supported by Canonical, should we continue to support Node 6 on it?
+ 
+@mhdawson: If we tell people from the beginning of the semver-major that support for unsupported platforms will cease even within semver-major cycle, that would be okay.
+ 
+@mhdawson: Recommended this text: "Community doesn't build and test against EOL distributions..."
```
Comment (mhdawson) [inline meetings/2017-06-07.md:61]: doesn't -> does not

File: meetings/2017-06-07.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+ 
+@rvagg: in practice we support libc and compiler (e.g. gcc) versions. As long as supported versions of those are on the target system it should work. So we may never run into problems here as long as we're supporting that rough environment.
+ 
+@mylesborins: any objections to dropping EOL OS's from CI? @rvagg: Binaries for later versions are sometimes built on earlier versions, e.g. CentOS 5 for Node < 8.
+ 
+@mhdawson: supported kernel levels don't change for a semver-major release.
+ 
+@mylesborins: maybe we should deliberately choose kernel/libc/gcc versions before we go LTS.
+ 
+@mhdawson: we did this for 4 for AIX.
```
Comment (mhdawson) [inline meetings/2017-06-07.md:87]: 4 -> 8

**Review by mhdawson — APPROVED** [review]
LGTM once my suggested updates are incorporated.

**Comment by Trott** [comment]
Made @mhdawson's requested changes and landed in e76909d7ca91f3e0ebad759a104368c3ae5be41d

**Pull request closed without merging**